config.rdf

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="owl2html.xsl"?>
+<rdf:RDF 
+     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+     xmlns:owl="http://www.w3.org/2002/07/owl#"
+     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+
+    <owl:Ontology rdf:about="http://fedora.info/definitions/v4/config#">
+        <rdfs:label xml:lang="en">Fedora Commons Configuration Ontology</rdfs:label>
+        <rdfs:comment xml:lang="en">Ontology for user-defined configuration properties and classes on Fedora-curated resources.</rdfs:comment>
+    </owl:Ontology>
+
+    <owl:DatatypeProperty rdf:about="http://fedora.info/definitions/v4/config#hasIndexingTransformation">
+        <rdfs:label xml:lang="en">has indexing transformation</rdfs:label>
+        <rdfs:domain rdf:resource="http://fcrepo.info/definitions/v4/repository#Container"/>
```

Comment:
Well, please don't decide on the single namespace approach just because there is an existing pull request. The associated code changes in fcrepo-camel and fcrepo-message-consumer are truly minimal.

I would only add that both Stefano and I have use cases for the *:Indexable class and *:hasIndexingTransform property, and the PR was primarily an attempt to begin a discussion so that we can move things along.

@ajs6f I am not particularly familiar with the use of the [owl:imports](http://www.w3.org/TR/owl-ref/#imports-def) statement. If A imports B, does that mean that all of the classes and properties of B are now available under the namespace of A?